pkg/hocon/hocon.go

```diff
@@ -46,13 +46,10 @@ func (k *KiltHocon) prepareFullStringConfig(info *kilt.TargetInfo) (*configurati
 		return nil, fmt.Errorf("could not serialize info: %w", err)
 	}
 
-	fmt.Println("ANTANI", string(rawVars), "ANTANI")
```

Comment:
these were debugging outputs that somehow ended in the repo from a previous commit.